decimal.go

```diff
@@ -423,6 +423,28 @@ func (d Decimal) Equals(d2 Decimal) bool {
 	return d.Equal(d2)
 }
 
+// Greater Than (GT) returns true when d is greater than d2.
+func (d Decimal) GT(d2 Decimal) bool {
```

Comment:
Changes look good overall, my main nitpick is with the method names. 

I would prefer them to be more explicit `GreaterThan`, `GreaterThanOrEqual`, `LessThan`, and `LessThanOrEqual`.

Out of context it's fairly difficult to know what `number.LT(otherNumber)` would do at a glance.